File: mobile/src/main/java/com/hotpodata/redchain/adapter/viewholder/RowTextOneLineViewHolder.java
package com.hotpodata.redchain.adapter.viewholder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

/**
 * Created by jdrotos on 1/21/15.
 */
public class RowTextOneLineViewHolder extends RecyclerView.ViewHolder {
    public TextView mTextOne;
    public ImageView mIcon;

    public RowTextOneLineViewHolder(View itemView) {
        super(itemView);
        mTextOne = (TextView) itemView.findViewById(android.R.id.text1);
        mIcon = (ImageView) itemView.findViewById(android.R.id.icon);
    }
}

File: mobile/src/main/java/com/hotpodata/redchain/adapter/viewholder/RowTextOneLineViewHolder.kt
package com.hotpodata.redchain.adapter.viewholder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView

/**
 * Created by jdrotos on 1/21/15.
 */
class RowTextOneLineViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var mTextOne: TextView
    var mIcon: ImageView

    init {
        mTextOne = itemView.findViewById(android.R.id.text1) as TextView
        mIcon = itemView.findViewById(android.R.id.icon) as ImageView
    }
}
